packages/tasks/src/model-libraries.ts

```diff
@@ -662,6 +662,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"tokenizer-e351c8d8-checkpoint125.safetensors"`,
 	},
+	mtvcraft: {
+		prettyLabel: "MTVCraft",
+		repoName: "MTVCraft",
+		repoUrl: "https://github.com/baaivision/MTVCraft",
+		filter: false,
+		countDownloads: `path_extension:"pt"`,
```

Comment:
Better to count on a single file to avoid duplicate counts. Typically on a file that you **have to** download to use the model. Would [./vae/3d-vae.pt](https://huggingface.co/BAAI/MTVCraft/blob/main/vae/3d-vae.pt) be a good candidate for instance?


```suggestion
		countDownloads: `path:"vae/3d-vae.pt"`,
```